### PR TITLE
168367747 dynamic db feature flag for sea route maintenance mode

### PR DESCRIPTION
**Pull request by philosotractor**
# Added
* language: error landing page and maintenance break strings
* services
routes: :maintenance-break-sea-route flag checked from db
routes: endpoints wrapped to service state check handler
* db
cljc specql namespace for feature variation
create feature_variation table and feature_type
* views
route-wizard: load stops failure handler to common servererror
route-list: load routes failure handler to common servererror
main: error-landing page selection
error-landing: first WIP version
 ote.sovellus height to 100vh to fill screen and put footer down
* controller: common: ServerError to redirect 503 to error-landing page
* ote: revert lein-figwheel to 0.5.13, connecting to browser not ok

### CODE REVIEW

**Comment by philosotractor** [comment]
Probably `no-cache-transit-response` should be used used in route endpoint, will improve that in following PR

File: ote/src/clj/ote/services/routes.clj [hunk 1]
```diff
@@ -19,11 +19,21 @@
...
             [ote.db.tx :as tx]
             [jeesql.core :refer [defqueries]]
-            [ote.environment :as environment])
+            [ote.environment :as environment]
+            [ote.db.feature])                               ; specql table definitions
   (:import (org.postgis PGgeometry Point Geometry)))
 
 (defqueries "ote/services/routes.sql")
 
+(defn- service-state-response [db]
```
Comment (rckopteri) [inline ote/src/clj/ote/services/routes.clj]: It is private function and only related to this service but still wondering if it should be renamed something like 'valid-route-service-state?'

File: ote/src/cljc/ote/db/feature.cljc [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "Data model for dynamic feature flags"
+  (:require
+            #?@(:clj [[ote.db.specql-db :refer [define-tables]]
+                      [specql.postgis]
+                      [clj-time.coerce :as tc]]
+                :cljs [[cljs-time.coerce :as tc]])
+            [specql.impl.registry]
+            [specql.data-types]
+            [ote.db.common]
+            ;[ote.db.modification]
```
Comment (rckopteri) [inline ote/src/cljc/ote/db/feature.cljc]: comment

File: ote/src/cljs/ote/views/error_landing.cljs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+(ns ote.views.error-landing
```
Comment (rckopteri) [inline ote/src/cljs/ote/views/error_landing.cljs]: Consider moving to own package like errors or somethin. ote.views.errors.error-landing

File: ote/src/clj/ote/services/routes.clj (hunk 1 above)
Comment (philosotractor) [reply ote/src/clj/ote/services/routes.clj]: I see what your going for. However,
Logic is naming is the same as in admin service side: function returns a request response, hence name should have "-response" suffix. "get-route-response" in the file follows the same pattern.
Suffix "?" is for boolean return values.

File: ote/src/cljs/ote/views/error_landing.cljs (hunk 3 above)
Comment (philosotractor) [reply ote/src/cljs/ote/views/error_landing.cljs]: Ok...what role and purpose that package would serve, in addition to the error landing page?

Comment (rckopteri) [reply ote/src/cljs/ote/views/error_landing.cljs]: We have quite many files in the views folder/package. We should split them to sub folders to help us navigate and find relevant files.

e.g. we have 2 at least 5 files that relates to transport-service and they are in alphabetical order. If they were in single sub folder finding them would be faster.

And from this point of view I'm expecting that we will have more than one error page in a few months so I would suggest that we add one folder for those :)

Comment (philosotractor) [reply ote/src/cljs/ote/views/error_landing.cljs]: As discussed, design is to have just one reused error_landing view and file. Nevertheless I moved this to new package.

**Pull request merged**